### PR TITLE
chore: Update LoadingDots styles

### PR DESCRIPTION
**Pull request by RayRedGoose**
## Summary

Resolves: #2545 

`LoadingDots` has been updated to use system color tokens and stencils.

## Release Category
Components

### CODE REVIEW

File: modules/react/loading-dots/lib/LoadingDots.tsx [hunk 1]
```diff
@@ -20,41 +15,48 @@ const keyframesLoading = keyframes({
...
-    animationDelay: '0ms',
-  },
-  '&:nth-child(2)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (4/35))`, // 160
-  },
-  '&:nth-child(3)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (8/35))`, // 320
+const loadingDotStencil = createStencil({
+  vars: {
+    animationDurationMs: '40ms',
```
Comment (mannycarrera4) [inline modules/react/loading-dots/lib/LoadingDots.tsx:20]: we don't have tokens for animation or `duration` right?

Comment (josh-bagwell) [reply modules/react/loading-dots/lib/LoadingDots.tsx:20]: I don't think we do have those tokens

Comment (RayRedGoose) [reply modules/react/loading-dots/lib/LoadingDots.tsx:20]: no, we don't. I don't think we have it in our roadmap soon

File: modules/react/loading-dots/lib/LoadingDots.tsx [hunk 2]
```diff
@@ -20,41 +15,48 @@ const keyframesLoading = keyframes({
...
-const LoadingAnimationDot = () => <div className={`${loadingDotStyles}`}></div>;
+const LoadingAnimationDot = () => <div {...loadingDotStencil()} />;
 
 /**
  * A simple container for the loading dots.
  */
-const containerStyles = createStyles({
-  display: 'inline-flex',
-  gap: system.space.x2,
+const loadingDotsStencil = createStencil({
```
[the remark this answers is not in the document]
Comment (RayRedGoose) [reply modules/react/loading-dots/lib/LoadingDots.tsx:55]: yeah, I agree. I like your idea about `singleLoadingDotStencil` name

File: modules/react/loading-dots/lib/LoadingDots.tsx [hunk 3]
```diff
@@ -20,41 +15,48 @@ const keyframesLoading = keyframes({
...
-  animationFillMode: 'both',
-  '&:nth-child(1)': {
-    animationDelay: '0ms',
-  },
-  '&:nth-child(2)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (4/35))`, // 160
-  },
-  '&:nth-child(3)': {
-    animationDelay: `calc(${ANIMATION_DURATION_MS}ms * (8/35))`, // 320
+const loadingDotStencil = createStencil({
```
Comment (RayRedGoose) [inline modules/react/loading-dots/lib/LoadingDots.tsx:18]:
```suggestion
const singleLoadingDotStencil = createStencil({
```

File: modules/react/loading-dots/lib/LoadingDots.tsx [hunk 4]
```diff
@@ -20,41 +15,48 @@ const keyframesLoading = keyframes({
...
+      animationDelay: calc.multiply(animationDurationMs, 8),
+    },
+  }),
 });
 
 /**
  * The actual loading dot div.
  */
-const LoadingAnimationDot = () => <div className={`${loadingDotStyles}`}></div>;
+const LoadingAnimationDot = () => <div {...loadingDotStencil()} />;
```
Comment (RayRedGoose) [inline modules/react/loading-dots/lib/LoadingDots.tsx:50]:
```suggestion
const LoadingAnimationDot = () => <div {... singleLoadingDotStencil()} />;
```

File: modules/react/loading-dots/lib/LoadingDots.tsx (hunk 1 above)
Comment (mannycarrera4) [reply modules/react/loading-dots/lib/LoadingDots.tsx:20]: all good I was just curious if that becomes some type of token? Maybe a larger discussion around animation!

**Review by mannycarrera4 — APPROVED** [review]

**Pull request merged**